gating/post_merge_test/run

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -xeu
+
+# This script prepares the lint environment in the same way the
+# docker file does, but without the container as its running on
+# single use node.
+
+sudo apt-get update && sudo apt-get install -y \
+  groovy2 python-pip build-essential python-dev libssl-dev \
+  curl libffi-dev sudo git-core
+
+pip install -c constraints.txt -r requirements.txt
+pip install -c constraints.txt -r test-requirements.txt
+
+RPC_GATING_LINT_USE_VENV=no ./lint.sh
```

Comment:
I'm probably missing the obvious here, but I thought `gating/pre_merge_test/run` in rpc-gating was to essentially replace the Single-Use-Slave-Example job.  Why are we doing a lint test here?

Reply:
I think this is just a demonstration of some sort of job, right @hughsaunders ? The idea being that each job type is exercised in rpc-gating.

Reply:
Yeah the idea is that the hook is for a periodic job. Rpc-gating doesn't really have many tests so I just reused the lint script.

Reply:
Fair enough!